apps/site/src/pages/api/vote-application.api.ts

```diff
@@ -1,11 +1,42 @@
+import Ajv from "ajv";
 import { createBaseHandler } from "../../lib/api/handler/base-handler";
 import { getMember, subscribeToMailchimp } from "../../lib/api/mailchimp";
 
 export type VoteApplicationRequestBody = {
   email: string;
-  merge_fields?: { [key: string]: any };
+  merge_fields: {
+    ECO_SANITY?: string;
+    ECO_STRAPI?: string;
+    ECO_CONFUL?: string;
+    ECO_GITHUB?: string;
+    ECO_OTHER?: string;
+    WISH_EA?: string;
+  };
 };
 
+const ajv = new Ajv();
+
+const validate = ajv.compile<VoteApplicationRequestBody>({
+  type: "object",
+  properties: {
+    email: { type: "string" },
+    merge_fields: {
+      type: "object",
+      properties: {
+        ECO_SANITY: { type: "string" },
+        ECO_STRAPI: { type: "string" },
+        ECO_CONFUL: { type: "string" },
+        ECO_GITHUB: { type: "string" },
+        ECO_OTHER: { type: "string" },
```

Comment:
If we don't rely on `additionalProperties`, we can validate these fields even further

```suggestion
        ECO_SANITY: { type: "string", enum: ["Yes", "No"] },
        ECO_STRAPI: { type: "string", enum: ["Yes", "No"] },
        ECO_CONFUL: { type: "string", enum: ["Yes", "No"] },
        ECO_GITHUB: { type: "string", enum: ["Yes", "No"] },
        ECO_OTHER: { type: "string", enum: ["Yes", "No"] },
```